libcxx/include/__format/buffer.h

```diff
@@ -115,7 +115,7 @@ class _LIBCPP_TEMPLATE_VIS __output_buffer {
 
     // The output doesn't fit in the internal buffer.
     // Copy the data in "__capacity_" sized chunks.
-    _LIBCPP_ASSERT_UNCATEGORIZED(__size_ == 0, "the buffer should be flushed by __flush_on_overflow");
+    _LIBCPP_ASSERT_INTERNAL(__size_ == 0, "the buffer should be flushed by __flush_on_overflow");
```

Comment:
@mordante If you have the time, it would be great if you could see if the classification in `<format>` and `<charconv>` looks reasonable to you. I classified many of these as "internal" because, if I'm reading the code correctly, these check for internal variants that should always hold unless there's a bug in our implementation. But I'm not very familiar with this part of the code and would really appreciate your feedback. Thanks!